src/IECoreRI/NSI/AttributeAlgo.cpp

```diff
@@ -50,6 +50,7 @@ NSIType_t convertGeometricInterpretation( GeometricData::Interpretation interpre
 	switch( interpretation )
 	{
 		case GeometricData::None :
+		case GeometricData::UV :
```

Comment:
This isn't right, because NSITypeVector implies 3 floats, whereas UV implies 2. In practice it won't matter because this function isn't called for non-V3f data, but just returning NSITypeInvalid for UV might make any future problems easy to diagnose...

Reply:
Addressed in 5bec0cf799fa89ba7a3bf27c33c3047924185653